ui/components/app/permissions-connect-permission-list/index.scss

```diff
@@ -0,0 +1,22 @@
+.permissions-connect-permission-list {
+  .permission {
+    @include H6;
+
+    width: 100%;
+    padding-bottom: 16px;
```

Comment:
Not a big deal but if this was a reusable component I would suggest any outside spacing to be added at the page level or where the layout requires it using a className.

Reply:
It's internal spacing - the 1px bottom border is part of the item